src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -85,6 +86,7 @@ GeantPhysicsList::GeantPhysicsList(Options const& options) : options_(options)
         em_parameters.SetMscEnergyLimit(100 * CLHEP::TeV);
     }
     em_parameters.SetApplyCuts(options.apply_cuts);
+    this->SetDefaultCutValue(options.default_cutoff * CLHEP::cm);
```

Comment:
Wouldn't this forcefully override any cutoff value defined originally by the gdml?

Reply:
I'm not positive about that... but at least I don't think it will change the current behavior: `G4RunManagerKernel::InitializePhysics()` calls `physicsList->SetCuts()`. The default `SetCuts()` method in `G4VUserPhysicsList` calls `SetDefaultCutValue()`, so since we don't have our own `SetCuts()` method it will be called either way. So all this *should* do is possibly change the default from 1 mm to something else.

Reply:
So that means that we have to add the `SetCuts()` correctly in the near future, when we actually start using the secondary production thresholds correctly.